relay-spans/src/span.rs

```diff
@@ -148,10 +169,16 @@ pub fn otel_to_sentry_span(otel_span: OtelSpan) -> EventSpan {
     // TODO: This is wrong, a segment could still have a parent in the trace.
     let is_segment = parent_span_id.is_empty().into();
 
+    if let Some(http_method) = http_method {
+        if let Some(http_route) = http_route {
+            description = format!("{} {}", http_method, http_route);
+        }
+    }
```

Comment:
You're updating the description for `db` in the loop, but http outside?

Reply:
because it relies on two different attributes